src/os/config/displaysettings.js

```diff
@@ -431,6 +459,55 @@ os.config.DisplaySettingsCtrl.prototype.getZoom = function() {
 };
 
 
+/**
+ * Get some info about the renderers for display purposes
+ * @return {Array<Object<string, string>>}
+ * @private
+ */
+os.config.DisplaySettingsCtrl.prototype.parseRenderers_ = function() {
+  const retArr = [];
+  const renderers = os.MapContainer.getInstance().getWebGLRenderers();
+  if (renderers) {
+    for (const [key, val] of Object.entries(renderers)) {
+      const temp = {};
+      temp['id'] = key == val.id ? key : os.config.DisplaySettingsCtrl.DEFAULT_RENDERER_ID;
+      temp['label'] = temp['id'].charAt(0).toUpperCase() + temp['id'].substr(1);
```

Comment:
While this works through convention, it may be better to allow the renderer to provide its own user-facing label along with the description added in this PR.